UACloudLibraryGraphQLClient/UACloudLibClientLibrary/RestClient.cs

```diff
@@ -104,6 +105,24 @@ public async Task<AddressSpace> DownloadNodeset(string identifier)
             return resultType;
         }
 
+        public async Task<(string,string)[]> GetNamespacesAsync()

```

Comment:
I vouch for using named tuples wherever we use tuples.

Reply:
Agreed. Fixed.